x-pack/plugins/security_solution/public/detections/components/rules/step_about_rule/index.tsx

```diff
@@ -438,12 +361,21 @@ const StepAboutRuleComponent: FC<StepAboutRuleProps> = ({
           </EuiAccordion>
         </Form>
       </StepContentWrapper>
-
-      {!isUpdateView && (
-        <NextStep dataTestSubj="about-continue" onClick={handleSubmit} isDisabled={isLoading} />
-      )}
     </>
   );
 };
 
 export const StepAboutRule = memo(StepAboutRuleComponent);
+
+const StepAboutRuleReadOnlyComponent: FC<StepAboutRuleReadOnlyProps> = ({
+  addPadding,
+  defaultValues: data,
+  descriptionColumns,
+}) => {
+  return (
+    <StepContentWrapper data-test-subj="aboutStep" addPadding={addPadding}>
+      <StepRuleDescription columns={descriptionColumns} schema={defaultSchema} data={data} />
+    </StepContentWrapper>
+  );
+};
+export const StepAboutRuleReadOnly = memo(StepAboutRuleReadOnlyComponent);
```

Comment:
++ I think only a shallow comparison is being made, so it the props may nullify the memo each time.

Reply:
Yeah I haven't checked to see if we're getting a benefit here or not - the forms lib doesn't seem great about providing the same object on each render so `defaultValues` might be changing a lot and taking away the memo benefits. But, the "Read only" view was memoized before when it was combined with the edit component, so I just memoized it here to keep the behavior as similar as possible.

I have another [branch](https://github.com/elastic/kibana/compare/main...marshallmain:kibana:create-rules-refactor) where I was working on improving the memoization around the create rules page, but I wasn't seeing a noticeable difference in user experience so I shelved it to work on this refactor instead. But, it could be worth picking it up again next.